### ISSUE TITLE
Configure Initial Setup for SAP Mobile Cards

### ISSUE THREAD

**Issue by jakes2255** [SAPDocuments/Tutorials#3982]
Tutorial URL: https://developers.sap.com/tutorials/cp-starter-digitalexp-mobilecards-1-setup.html

Please specify the step you are referring to:Step 2
Hi SAP Team,
Under Step2, " Mobile Services cockpit " shows a Message that "
Mobile Services is not enabled. The service is required for upgrade. Check SAP Help Portal for instructions." 
Further again in Ste2,  at sub step(9), when I switch back to "Mobile Services cockpit', there is no popup being shown asking me to enter the API Key.

What could be the issue? Could you please check?

Thank you,
Jakes

**Comment by jitendrakansal** [comment]
@jakes2255 
Regarding warning message "Mobile Services is not enabled" -  this is due to there is new release of mobile services rolled out in CF recently. 

you may check details here: https://help.sap.com/viewer/468990a67780424a9e66eb096d4345bb/Cloud/en-US/a27b7a29aeec4201b7faf684e93c71df.html

For another issue - regarding pop up..
Could you share details? When you navigated to "SAP Mobile Cards" tab, Had you already initialized mobile cards instance? 
Could you also check in SAP Mobile Cards > Features > Mobile Connectivity , if you already have SFDEMO_APIHUB destination.

**Comment by jakes2255** [comment]
Thank you Jithendra,

Yes, I had ignored the warning message and proceeded to complete the  steps as mentioned in the tutorial.

Regarding the second issue:-
Still not getting the popup as mentioned at "_Switch back to Mobile Services cockpit_"
![2019_10_02_21_57_20_SAP_Cloud_Platform_Mobile_Services_Trial](https://user-images.githubusercontent.com/40160913/66062907-ecdbea80-e55f-11e9-9942-03435afc8423.png)

What could be the issue? 
Is this not available on the trial users on cf?
Please see when you have some time.

Thanks,
Jakes

**Comment by jitendrakansal** [comment]
@jakes2255 
Tutorial you are following is created with CF trial only.

You need to create the destination first. 
Follow step here https://developers.sap.com/tutorials/cp-starter-digitalexp-mobilecards-1-setup.html#e0acd616-fe0b-4bec-8189-e2533d08f1ea
You need to navigate to template manager, select any SFSF template, click create. It would ask you to enter API key that can be retrieved from api.sap.com as per instructions. Once you provide this key, you should see "destination" creation message. 
Pls give it a try and keep me posted.

**Comment by jakes2255** [comment]
Thank you Jitendra,

I am also trying the same in CF trial.

However when trying to create the SF destination, there is no popup asking for API key is coming.

Thank you,
Jakes

**Comment by jitendrakansal** [comment]
@jakes2255 
maybe we can check it together, what about a short call in next 30 mins? Are you available? pls drop me a note on my email address
jitendra.kansal@sapdotcom

**Comment by jakes2255** [comment]
Sorry Jithendra, I saw your response late.

I have sent you a meeting invite for 4pm today, please revert if you are not available on that time.

Thank you,
Jakes

**Comment by thecodester** [comment]
Now that this is taken offline, I am closing the issue. But @jitendrakansal if you want to describe how you resolved the issue, feel free to add a comment.

**Comment by jakes2255** [comment]
Thank you Jitendra for resolving both the issues👍👍👍
1) For the first upgrade was performed.
2) Pop up started appearing when attempted with URL upto ....index.html on the cockpit.

Also many thanks for taking out through the upcoming "Recipe" based development as well.
Thanks a lot.👍👍👍

Kind Regards,
Jakes

**Comment by jitendrakansal** [comment]
Glad that i was able to help. Thanks.

**Comment by mukhutis** [comment]
@jakes2255, I am having the exact same problem. No popup for API key after I click the "Create" button against the SF template - in this case, the timesheet one. I have the API key, but nowhere to paste.

Were you able to resolve this?

**Comment by jakes2255** [comment]
Hi @mukhutis ,

Yes we were able to get this resolved.

Please copy the URL upto ".....index.html" and then open the link in Chrome Incognito window.

Are you able to proceed?
Are you getting the API pop up.

If not, are you getting a warning for upgrade?

Thanks,
Jakes

**Comment by mukhutis** [comment]
Hi @jakes2255,

Problem resolved, but the solution was somewhat different.

1. I start with URL up to "...index.html" and open in Chrome incognito, it takes me to the SAP Mobile Cards Home Page.

2. Then when I navigate to Mobile Applications -> SAP Mobile Cards -> Template Manager, it adds the hashed navigation "...index.html#/page.view.sapcards/com.sap.mobile.platform.cards" 

3. At this stage, I removed the portion after index.html but did not press Go, since it will take me to step 1 (home page)

4. as soon as I click create button, the hashed navigation changes to "../index.html#/page.create.sapcards" with no pop-ups to add API key or warning to upgrade.

I did saw something popping up momentarily before disappearing despite popup blockers being disabled. I tried a different browser as well with no luck.

Then I tried the same steps in incognito but **with a different SuccessFactors card** than the one I was trying. This time, the pop up came and the destination was added.

Now I am able to find the SFDEMO_APIHUB destination in all other SuccessFactors card as well including the original card. Problem solved!

Thanks 
Subh
